src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquirer.cs

```diff
@@ -84,6 +84,7 @@ async Task<AcquireTokenResult> ITokenAcquirer.GetTokenForAppAsync(string scope,
                 PopPublicKey = tokenAcquisitionOptions.PopPublicKey,
                 PopClaim = tokenAcquisitionOptions.PopClaim,
                 ExtraParameters = tokenAcquisitionOptions.ExtraParameters,
+                FmiPath = tokenAcquisitionOptions.FmiPath
```

Comment:
Is there any test you can write that uses reflection to protect against this kind of developer error? Smth like checking the names of properties between `AcquireTokenOptions` and `TokenAcquisitionOptions` ?